recipes-connectivity/bluez5/bluez5_%.bbappend

```diff
@@ -1,6 +1,6 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 
-SRC_URI_append_raspberrypi3 = " \
+SRC_URI_append = " \
```

Comment:
if they should apply to all rpi SOCs then use `_rpi` override please. We can not add patches unconditionally since this will mess up other BSPs when doing multi machine builds.